react-form-controls/my-app/src/RegistrationFormUncontrolled.jsx

```diff
@@ -0,0 +1,28 @@
+export default function RegistrationFormUncontrolled() {
+  function handleSubmit(event) {
```

Comment:
Please review step 5 of the exercise and refactor with the below comments. Let us know if you have any questions in slack!